starter-code/css/main.css

```diff
@@ -0,0 +1,66 @@
+* {
+  margin: 10px;
+  font-family: 'Lato', sans-serif;
+  text-align: left;
+}
+body {
+  text-align: left;
+}
+h1 {
+  font-size: 45px;
+  text-align: left;
+  font-weight: 100;
+  margin-left: 0;
+}
+h2 {
+  font-size: 30px;
+  font-weight: lighter;
+  letter-spacing: -1px;
+}
+h2 a {
+  color: #000000;
+  font-weight: 550 !important;
```

Comment:
You want to try and avoid using `!important` when coding in CSS. Usually when you feel like you have to use it, it means there is something overriding it somewhere else that should be corrected. The font-weight here is set to 550 but the google font that is being used didn't have that as a font-weight option. Make sure the fonts support the correct font-weight values.